### PR TITLE
Fix uniqueness for events uid

### PR DESCRIPTION
**Pull request by 0lha**

### CODE REVIEW

File: gamma_bridge/exceptions.py [hunk 1]
```diff
@@ -40,3 +40,13 @@ class NotBaseStorageError(GammaValueError):
     def __init__(self, message=None, *args):
         self.message = "The value type must be BaseStorage. {}".format(message)
         super(GammaException, self).__init__(self.message, *args)
+
+
+class GammaEventDataError(GammaException):
+    """
+    Error is occurred when tracking event don't contain data, required to build uid
```
Comment (cmltaWt0) [inline gamma_bridge/exceptions.py]: Add dot at the end please.

Comment (cmltaWt0) [reply gamma_bridge/exceptions.py]: Also typo in don't.
Should be `Error is occurred when tracking event doesn't contain data, required to build uid`.
Also keep in mind that docstring can't exceed 80 chars.

**Review by cmltaWt0 — CHANGES_REQUESTED** [review]
Small typo and  let's merge this.

File: gamma_bridge/exceptions.py (hunk 1 above)
Comment (0lha) [reply gamma_bridge/exceptions.py]: done

**Review by cmltaWt0 — APPROVED** [review]

**Pull request merged**